pkg/asset/installconfig/ibmcloud/validation_test.go

```diff
@@ -1,24 +1,30 @@
-package ibmcloud_test
```

Comment:
This was an easy way to avoid moving types

Reply:
I believe there may be a way to avoid this and keep types within the ibmcloud package using some genmock magic